multimedia/mlt/Portfile

```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           muniversal 1.0
 PortGroup           github 1.0
+PortGroup           cmake 1.1
 PortGroup           active_variants 1.1
 PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        mltframework mlt 6.24.0 v
+github.setup        mltframework mlt 7.22.0 v
```

Comment:
FYI today I will be releasing a new version 7.24.0

Reply:
Vvrsion 7.24.0 released: https://github.com/mltframework/mlt/releases/tag/v7.24.0
with checksums:
```
rmd160 6efa1a3daf2ae3c2aa3a3714c43df49901f23895
sha256 8cde7c22a1a5395abe913976c2edafb498f81ed81a5f49dd0e6e2d86d68bcec0
size 1662552
```